docpages/01_frequently_asked_questions.md

```diff
@@ -1,88 +1,113 @@
+
+The easiest way is to use our [template project](https://github.com/brainboxdotcc/windows-bot-template). If you are unable to do this, download the precompiled latest release from our GitHub releases, and take the dlls, `.lib` file, and header files (`bin`, `lib` and `include` directories), placing them in a easily accessible place on your computer. Go into Visual Studio project settings in a new project, and point the project directories (notably the library directories and and include directories) at the correct locations. Add the `include` folder you extracted to your include directories, and add `dpp.lib` to your library directories. Ensure the project is set to C++17 standard or later in the settings. You should then be able to compile example programs within that project. When you run the program you have compiled you must ensure that all the dll files from the `bin` directory exist in the same directory as your executable.
 
 ## Does this library support Visual Studio 2022?
+
 Yes! The master branch comes with pre-built binaries for Visual Studio 2022 and our Windows bot template has a [vs2022 branch](https://github.com/brainboxdotcc/windows-bot-template/tree/vs2022) which you can clone to get Visual Studio 2022 specific code. For the time being we support both Visual Studio 2019 and 2022. At some point in the future only 2022 may be supported as 2019 becomes outdated.
 
 ## How much of the library is completed?
+
 All REST calls (outbound commands) are completed including all currently available interactions, and all Discord events are available. The library also has voice support.
 
 ## How do I chat with the developers or get help?
+
 The best place to do this is on the [Discord server](https://discord.gg/dpp). You most likely won't get an answer immediately (we have lives, and need to sleep sometimes), but we will be willing to help!
 
 ## How can I contribute to development?
+
 Just star and fork a copy of the repository, and submit a Pull Request! We won't bite! Authors of accepted pull requests get a special role on our [Discord server](https://discord.gg/dpp).
 
 ## What's the best way to learn C++?
+
 A simple search can find some learning tools, however not all are good. Here is a list of some (good) learning resources:
 
-* [CodeAcademy](https://www.codecademy.com/learn/c-plus-plus)
+* [Codecademy](https://www.codecademy.com/learn/c-plus-plus)
 * [Learn CPP](https://www.learncpp.com/)
 * [Learn CPP (Very Basic)](https://www.learn-cpp.org/)
 
-If you don't understand something then feel free to ask in the [Discord server](https://discord.gg/dpp) ...*we don't bite!*
+We do ask that you don't come into the [Discord server](https://discord.gg/dpp) and expect help for everything C++!
```

Comment:
expect help for everything c++? hmm dunno, not homework etc. I like the old and more casual statement here

Reply:
This says that we won't help for everything C++ (meaning we won't be helping with general c++ stuff)

Reply:
I think the wording is confusing. the older one with we don't bite is much more friendly

Reply:
Yeah, I've reverted it back :)